client/pages/section.js

```diff
@@ -5,10 +5,27 @@ import { indexedDBSync, ajaxSync } from '../actions/projects';
 import DefaultSection from './sections';
 import SectionsLink from '../components/sections-link';
 import ErrorBoundary from '../components/error-boundary';
+import StaticSection from '../components/static-section';
 import Readonly from './readonly';
 import { getSubsections } from '../schema';
 
-const mapStateToProps = ({ project, application: { schemaVersion, readonly, establishment, isGranted } }, { match: { params } }) => {
+const mapStateToProps = (
+  {
+    project,
+    application: {
+      schemaVersion,
+      readonly,
+      establishment,
+      isGranted,
+      project: actualProject
```

Comment:
🤣

Reply:
Guess who's back, back again.

`actualProject` is back. Tell your friends.